### PR TITLE
Fix two VSIX issues stemming from Reunion -> WindowsAppSDK rename

### PR DESCRIPTION
**Pull request by DefaultRyan**
After the rename, it was decided to revert the VSIX id to the original
name so as not to break upgradability and continuity. Unfortunately,
the repositoryId elements in the individual templates weren't updated.

Additionally, the original rename introduced a typo in the XmlPoke build
target responsible for replacing the placeholder package versions with
the actual package versions.

### CODE REVIEW

File: dev/VSIX/Directory.Build.targets [hunk 1]
```diff
@@ -77,7 +77,7 @@
                 <OriginalNugetRepositoryId>$(_tempVsTemplateNugetRepositoryId)</OriginalNugetRepositoryId>
                 <!-- 'ExperimentalNugetRepositoryId' needs to match the value set by the XmlPoke query for
                 "/ns:PackageManifest/ns:Metadata/ns:Identity/@Id" in Extension\WindowsAppSDK.Extension.csproj -->
-                <ExperimentalNugetRepositoryId>$(_tempVsTemplateNugetRepositoryId).Experimental</ExperimentalNugetRepositoryId>
+                <ExperimentalNugetRepositoryId>$(_tempVsTemplateNugetRepositoryId).Preview</ExperimentalNugetRepositoryId>
```
Comment (kythant) [inline dev/VSIX/Directory.Build.targets:80]: Does this make it so the name of the vsix file now has preview in it instead of experimental?

Comment (DefaultRyan) [reply dev/VSIX/Directory.Build.targets:80]: I thought that VS used the VSIX Id, not the filename. I'm going to try testing it now.

Comment (alwu-msft) [reply dev/VSIX/Directory.Build.targets:80]: It needs to match the VSIX ID, as I noted in the comment. :)

**Comment by alwu-msft** [comment]
>             Query="/ns:VSTemplate/ns:TemplateContent/ns:CustomParameters/ns:CustomParameter[@Name='$WindowsAppSdkNupkgVersion$']/@Value"

This casing also needs to be fixed.

---
Refers to: dev/VSIX/Directory.Build.targets:218 in bade632. [](commit_id = bade63221ab759f110fcbde2f556daafba345557, deletion_comment = False)

**Comment by alwu-msft** [comment]
>             Query="/ns:VSTemplate/ns:TemplateContent/ns:CustomParameters/ns:CustomParameter[@Name='$WindowsAppSdkFoundationNupkgVersion$']/@Value"

This casing also needs to be fixed.

---
Refers to: dev/VSIX/Directory.Build.targets:221 in bade632. [](commit_id = bade63221ab759f110fcbde2f556daafba345557, deletion_comment = False)

**Comment by alwu-msft** [comment]
>             Query="/ns:VSTemplate/ns:TemplateContent/ns:CustomParameters/ns:CustomParameter[@Name='$WindowsAppSdkDWriteNupkgVersion$']/@Value"

This casing also needs to be fixed.

---
Refers to: dev/VSIX/Directory.Build.targets:224 in bade632. [](commit_id = bade63221ab759f110fcbde2f556daafba345557, deletion_comment = False)

**Comment by alwu-msft** [comment]
>             Query="/ns:VSTemplate/ns:TemplateContent/ns:CustomParameters/ns:CustomParameter[@Name='$WindowsAppSdkWinUINupkgVersion$']/@Value"

This casing also needs to be fixed.

---
Refers to: dev/VSIX/Directory.Build.targets:227 in bade632. [](commit_id = bade63221ab759f110fcbde2f556daafba345557, deletion_comment = False)

**Comment by DefaultRyan** [comment]
Fixed the cases mentioned by alwu.

**Comment by alwu-msft** [comment]
@DefaultRyan Please cherry-pick fe787c733230e758b3c17dc6c7ea2a5972c5dac9 ([https://github.com/microsoft/WindowsAppSDK/commit/fe787c733230e758b3c17dc6c7ea2a5972c5dac9](https://github.com/microsoft/WindowsAppSDK/commit/fe787c733230e758b3c17dc6c7ea2a5972c5dac9)) for the change the removes the Single-project MSIX Packaging project templates.

**Comment by huichen123** [comment]
we want this change in main and experimental. But your change may only for experimental.

---
In reply to: [894549399](https://github.com/microsoft/WindowsAppSDK/pull/1192#issuecomment-894549399) [](http://example.com/codeflow?ancestors=894549399,894549399)

**Comment by DefaultRyan** [comment]
> we want this change in main and experimental. But your change may only for experimental.
> 
> In reply to: [894549399](https://github.com/microsoft/WindowsAppSDK/pull/1192#issuecomment-894549399)

If it's broken, it's broken. In this case, I don't see much, if any benefit to isolating this change only to the experimental release branch.

**Review by alwu-msft — APPROVED** [review]
:shipit:

**Pull request merged**